src/data/i18n/locales/json/currentlySupportedLangs.jsx

```diff
@@ -14,3 +14,12 @@ import './zh_CN.json';
 //
 // Further, I hit the read replica in order to confirm the live values, which are currently:
 //    en, fr, zh-cn, es-419, ar
+
+const localeDataMap = {
+  'ar': arData,
+  'es-419': esData,
+  'fr': frData,
+  'zh-cn': zhData,
```

Comment:
I added the `quote-props` rule with `consistent-as-needed` to our `eslintrc` because of this block - otherwise it would have me leave `ar` and `fr` unquoted, but require quote around `es-419` and `zh-cn`, which seems silly and odd.